Actions/Github-Helper.psm1

```diff
@@ -1081,3 +1081,21 @@ function DownloadArtifact {
     InvokeWebRequest -Headers $headers -Uri $artifact.archive_download_url -OutFile $outFile
     $outFile
 }
+
+function Get-ActionOwner() {
+    if ($null -eq $ENV:GITHUB_ACTION_PATH) {
+        return ""
+    }
+
+    $ap = "$ENV:GITHUB_ACTION_PATH".Split('\')
```

Comment:
Comment to explain the assumptions around here, please.

Reply:
Removed